clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -12873,6 +12873,9 @@ def err_free_function_return_type : Error<
   "SYCL free function kernel should have return type 'void'">;
 def err_free_function_first_occurrence_missing_attr: Error<
   "the first occurrence of SYCL kernel free function should be declared with 'sycl-nd-range-kernel' or 'sycl-single-task-kernel' compile time properties">;
+def err_free_function_static_class_method : Error<
+  "static class method can not be used as SYCL kernel free function">;
```

Comment:
The messages are a bit funny now. Can we merge err_free_function_static_class_method and err_kern_is_nonstatic_method
and use a select
```suggestion
def err_free_function_static_class_method : Error<
  "%select{static |}0class method can not be used to define a SYCL kernel free function kernel">;
```

https://clang.llvm.org/docs/InternalsManual.html#the-format-string